beacon-chain/node/node.go

```diff
@@ -194,19 +196,49 @@ func (b *BeaconNode) Close() {
 }
 
 func (b *BeaconNode) startDB(ctx *cli.Context) error {
+	var clearDB bool
 	baseDir := ctx.GlobalString(cmd.DataDirFlag.Name)
 	dbPath := path.Join(baseDir, beaconChainDBName)
 	d, err := db.NewDB(dbPath)
 	if err != nil {
 		return err
 	}
 	if b.ctx.GlobalBool(cmd.ClearDB.Name) {
-		if err := d.ClearDB(); err != nil {
-			return err
+		reader := bufio.NewReader(os.Stdin)
```

Comment:
how about refactor this into its own function?  Something like `askToDelete` or better ?